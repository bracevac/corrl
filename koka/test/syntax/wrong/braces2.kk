function map(f,xs) {
  match(xs) {
    Nil -> Nil
    Cons(x,xx -> { val y = f(x); val yy = map(f,xx); Cons(y,yy) }
  }
}
