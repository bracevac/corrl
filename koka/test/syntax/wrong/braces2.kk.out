test/syntax/wrong/braces2.kk(4,15): error: invalid syntax
 unexpected keyword ->
 expecting "," or ")"